readme.md

```diff
@@ -88,6 +89,7 @@ It includes a command-line shell and an associated scripting language.
 ## Commandline Productivity
 
 * [posh-git](https://github.com/dahlbyk/posh-git) - Set of PowerShell scripts which provide Git/PowerShell integration.
+* [posh-ssh](https://github.com/darkoperator/Posh-SSH) - PowerShell Module for automating tasks on remote systems using SSH
```

Comment:
Win10 supports ssh.

Reply:
Although Win10 supports SSH, posh-ssh can be used on any .Net4.5 and PowerShell v3+ machine. We used it on a bunch of Windows Server 2008 R2 - thus it was suggested. Would you rather like to remove?